cli/disk_cache.rs

```diff
@@ -105,31 +105,25 @@ impl DiskCache {
 
         out = out.join(remaining_components);
       }
-      scheme => {
-        unimplemented!(
-          "Don't know how to create cache name for scheme: {}\n  Url: {}",
-          scheme,
-          url
-        );
-      }
+      _ => return None,
```

Comment:
Having this panic was causing other problems where we "assumed" the import specifier was valid, where we really check it in the file_fetcher.